<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExploreMap</class>
 <widget class="QDialog" name="ExploreMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Find The Shortest Path</string>
   </property>
   <widget class="QComboBox" name="city1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="city2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Second City</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findPath">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Find The Shortest Path</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="path">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>911</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>269</y>
     <width>1161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Traversal</string>
   </property>
   <widget class="QTextBrowser" name="traversal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="start">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BFS">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BFS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DFS">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DFS</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>419</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Insert City</string>
   </property>
   <widget class="QLineEdit" name="insertCity">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IC">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete City</string>
   </property>
   <widget class="QPushButton" name="DC">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>37</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DCity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>549</y>
     <width>671</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Insert Edge</string>
   </property>
   <widget class="QComboBox" name="IECity1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="IECity2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Second City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>41</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="distance">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="time">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="IE">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>549</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete Edge</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Second City</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DECity1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="DECity2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DE">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
